docs/News.md

```diff
@@ -1,3 +1,12 @@
+## October 9, 2019
+*  **Added data** consisting of  2615 samples from 3 studies:
+   
+     * [Cancer Cell Line Encyclopedia (Broad, 2019)](https://www.cbioportal.org/study/summary?id=ccle_broad_2019) *1739 samples*
```

Comment:
please replace the 2012 study with the 2019 study in the Cell lines section.

Reply:
@jj niki suggested keeping both and but add a note with the link to the new one

Reply:
sounds good